### PR TITLE
Fix Dark Mode Details

### PR DESCRIPTION
**Pull request by kaio-souza**
Fixes #

## Proposed Changes

1. Fix Icon Scenario
2. Fix Mermaid Arrows

SCENARIO BEFORE
scenario-before
SCENARIO AFTER
scenario-after

MERMAID ARROWS BEFORE
mermaid-before

MERMAID ARROWS AFTER
mermaid-after

### CODE REVIEW

File: .gitignore
Comment (dalssoft) [inline .gitignore:1]: not part of the proposed PR

File: .vscode/launch.json
Comment (dalssoft) [inline .vscode/launch.json:1]: not part of the proposed PR

**Review by dalssoft — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,11 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+# ide files
+.idea
+.vscode
+examples
+
```
Comment (kaio-souza) [inline .gitignore]:
```suggestion
```

**Review by dalssoft — APPROVED** [review]

**Comment by dalssoft** [comment]
"The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)"

**Pull request closed without merging**